src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+  width: 200px;
+  padding: 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  background: $white;
+  border: 1px solid $grayElements;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: auto;
+    margin-top: 16px;
```

Comment:
Will be better to specify correct top padding for the card instead of set extra margins for the image